sessions/recap-project-3/rick-and-morty-app/README.md

```diff
@@ -100,7 +98,9 @@ their name into the search bar.
 > 💡 You might run into some bugs at this point. Think about how the page and max page index might
 > have to change when you start searching for only subsets of all characters.
 
-### Bonus: Refactoring your Code
+## Bonus
```

Comment:
```suggestion
### Bonus: Refactoring your Code
```

Reply:
It was decided to keep the Bonus as h2.
Task are mandatory feature, meanwhile bonus and bonus, extra